src/main/java/org/zalando/nakadi/controller/ExceptionHandling.java

```diff
@@ -66,6 +70,12 @@ private String generateErrorTraceId() {
         return Responses.create(Response.Status.BAD_REQUEST, message, request);
     }
 
+    @ExceptionHandler(NoEventTypeException.class)
+    public ResponseEntity<Problem> noEventTypeException(final NoEventTypeException exception,
+                                                               final NativeWebRequest request) {
+        return Responses.create(UNPROCESSABLE_ENTITY, exception.getMessage(), request);
```

Comment:
As far as I see - this is class for default exception handlers. Maybe default one for `NoSuchEventType` is `NOT_FOUND`?